modules/admin_manual/pages/installation/linux_installation.adoc

```diff
@@ -3,11 +3,14 @@
 :apt-mark-hold-url: https://manpages.debian.org/stretch/apt/apt-mark.8.en.html#PREVENT_CHANGES_FOR_A_PACKAGE
 :yum-versionlock-plugin-url: http://man7.org/linux/man-pages/man1/yum-versionlock.1.html
 
-== Introduction
-
 NOTE: Package managers should only be used for single-server setups. For production environments, we recommend installing from
 https://owncloud.org/download/#owncloud-server-tar-ball[the tar archive].
 
+== Add the ownCloud Repository
+
+Before you can install `owncloud-files`, you need to add ownCloud's repository to your distribution's package manager.
+The instructions for doing so, are available https://download.owncloud.org/download/repositories/stable/owncloud/index.html[here].
+
```

Comment:
It would be good to move the note from above into this section because there is no introduction section anymore.

Reply:
Fair enough.